addOns/openapi/src/main/resources/org/zaproxy/zap/extension/openapi/resources/Messages.properties

```diff
@@ -31,24 +31,20 @@ openapi.formhandler.desc = OpenAPI Form Handler Integration
 openapi.formhandler.name = OpenAPI Form Handler
 
 openapi.import.error = Failed to access URL: {0} : {1} : {2}
-
-openapi.importfromdialog.importbutton = Import
-openapi.importfromdialog.invalidurl = Invalid URL:\n{0}
-openapi.importfromdialog.labelcontext = Context For Adding DDNs:
-openapi.importfromdialog.labeltarget = Target URL:
-openapi.importfromdialog.pasteaction = Paste
-
-openapi.importfromfiledialog.badfile = Can't find the specified file
-openapi.importfromfiledialog.choosefilebutton = Choose File
-openapi.importfromfiledialog.labelfile = File Path:
-openapi.importfromfiledialog.title = Import OpenAPI Definition from File
-
-openapi.importfromurldialog.labelurl = URL Pointing to OpenAPI defn:
-openapi.importfromurldialog.title = Import OpenAPI Definition from URL
-openapi.importfromurldialog.urlerror.empty = The URL to import is empty.
-openapi.importfromurldialog.urlerror.invalid = The URL to import is invalid:\n{0}
-
-openapi.io.error = Failed to access specified definition
+openapi.import.error.emptyDefn = The OpenAPI definition was null or empty.
```

Comment:
`… was empty.` for the user that it might be null is not relevant IMO (we also require the necessary data beforehand).